### PR TITLE
Some simpler edits

### PR DESCRIPTION
**Pull request by adrn**
Hey! Here are some of my simpler edits / suggestions. This is mostly small grammar / punctuation tweaks, but the one more substantial change I made was to reword some of the intro text about stars and heating. I think it's important to rephrase this to emphasize that the *observable* is that older stars have larger vertical velocities (or really a larger dispersion). This can either be interpreted as "stars are born young and get heated," or "older stars were born hot" - and I think the jury is still open on that. So I tried to rephrase to capture this, but take a look.

### CODE REVIEW

**Comment by adrn** [comment]
Oh wait, I just remembered one other "substantial" change buried in here! I added the word "Galactic" to the title - I think the word "kinematics" alone is maybe too generic?

**Comment by RuthAngus** [comment]
Yey, thank you so much!!! Dang, I love it when coauthors submit pull requests :-)
